Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.xaml

```diff
@@ -31,5 +31,7 @@
                 </ControlTemplate>
             </Setter.Value>
         </Setter>
+        <Setter Property="RotationDelay" Value="0:0:3" />
+        <Setter Property="ExtraRandomDuration" Value="0:0:10" />
```

Comment:
I think it was something that I changed for debug purposes and forgot it there. Thanks, modified with the latest commit.